docs/developer.md

```diff
@@ -109,6 +110,7 @@ python .\build\build.py `
   --cuda_version='10.1' `
   --cudnn_version='7.6.5'
 ```
+Discard `--cuda_compute_capabilities` flag in case of argument exception.
```

Comment:
I'd remove this comment. But instead I'd just remove `--cuda_compute_capabilities` from the command above. The defaults should work for most people, and even if they do not, it isn't a Windows-specific thing.

(It turns out we also accidentally removed the flag, but we'll add it back again.)

Reply:
That would be more straightforward. Thank you for your comment.